### DISCUSSION TITLE
Contributing guidelines

### DISCUSSION THREAD

**Discussion by BoomerScratch** [ScratchAddons/ScratchAddons#45] [Discussion]
For #20, we need to create a contributing guidelines. What should we write for the contributing guidelines?

**Comment by tb148** [comment]
Quoted from #20:
> Things I think worth including:
> 1. [Semantic Versioning](https://semver.org/)
> 2. [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)

Also I think we can create a draft poll request for that.

**Comment by WorldLanguages** [comment]
Not sure. Anyone should be able to contribute as long as they know git and JavaScript basics

**Comment by BoomerScratch** [comment]
> # Contributing guidelines
> For anyone who knows [Git](https://git-scm.com) and JavaScript basics: feel free to contribute to this repository! Our code is open source. Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/CODE_OF_CONDUCT.md).
>
> ## Filing [issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues)
> Found a bug? Report it! Be sure to check if it is already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
>
> Also, feel free to ask questions, suggest new features, and more.
> ## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
> Contributions are welcome! Be sure to file an issue first (see above). If you know a fix for that issue, or you know a fix for a different one, you can create a pull request. Fork this repository, create a new branch from the 'master' branch, and make your changes on the new branch. Now, create a pull request on the origin repository (ScratchAddons/ScratchAddons). We will review your pull request.

What do you guys think?

**Comment by WorldLanguages** [comment]
@BoomerScratch LGTM

**Comment by tb148** [comment]
Things I think worth adding:
1. [Semantic Versioning](https://semver.org/)
2. [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)

**Comment by WorldLanguages** [comment]
@tb148 they feel like too much for this project imo.
